app/controllers/application_controller.rb

```diff
@@ -6,11 +6,16 @@ class ApplicationController < ActionController::Base
   helper_method :current_user
   helper_method :google_maps_api_key
 
+  before_action :fake_ip unless Rails.env.production?
```

Comment:
Seems like we'd want to also allow for staging
`before_action :fake_ip unless Rails.env.production? || Rails.env.staging?`

Reply:
I think we might want to have a staging environment also show up as `Rails.env.production?`. At least, that's basically what Heroku recommends in terms of having different environments.

https://devcenter.heroku.com/articles/multiple-environments#managing-staging-and-production-configurations
https://devcenter.heroku.com/articles/deploying-to-a-custom-rails-environment#setting-staging-behavior